test.disabled/ejabberd_tests/tests/amp_big_SUITE.erl

```diff
@@ -279,6 +326,30 @@ notify_deliver_to_online_user_recipient_privacy_test(Config) ->
               client_receives_nothing(Bob)
       end).
 
+notify_deliver_to_online_user_broken_connection_test(Config) ->
+    escalus:fresh_story(
+      Config, [{alice, 1}, {bob, 1}],
+      fun(Alice, Bob) ->
+              %% given
+              Rule = {deliver, case ?config(offline_storage, Config) of
```

Comment:
Could you move the `case` to a separate function? It will improve readability.

Reply:
It's always a tradeoff between readability and conciseness. I could use the helper here, for me it makes no big difference though.